src/components/GlobalFooter/GlobalFooter.js

```diff
@@ -17,7 +18,8 @@ const GlobalFooter = () => {
     <footer className={styles.footerOuter}>
       <div className={styles.footerInner}>
         <div className={styles.logo}>
-          Logo
+          <img src={logo} alt = "logo-icon" />
```

Comment:
```suggestion
          <Link to={routes.HOME}>
            <img src={logo} alt="Logo-icon" />
            360&#176; Collaboration
          </Link>
```
You can wrap the `<img />` link with the `<Link>` component to make both the image and text a link

Reply:
Thank you very much - I couldn't figure out how to make that image clickable.